### PR TITLE
Change Default Button

### PR DESCRIPTION
**Pull request by annalena94**
I adjusted the colour of the Default Button

The Acceptance Criteria were: 
- Give Default Button the colour from the wireframes

Here's a screenshot:
image

### CODE REVIEW

File: Jointify/Assets.xcassets/ButtonColor.colorset/Contents.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (lgerhardt45) [inline Jointify/Assets.xcassets/ButtonColor.colorset/Contents.json:1]: Nice, I was wondering how you add custom colours to the whole project

File: Jointify/Views/Subviews/DefaultButton.swift [hunk 2]
```diff
@@ -46,3 +48,9 @@ struct DefaultButton_Previews: PreviewProvider {
     }
     
 }
+
```
Comment (lgerhardt45) [inline Jointify/Views/Subviews/DefaultButton.swift:51]:
```suggestion

// MARK: Extension Color
/// add the default colour for the button
```

Comment (lgerhardt45) [reply Jointify/Views/Subviews/DefaultButton.swift:51]: We have a style guide which I persistantly push and will send you just now :-). The MARKs are just a hygiene thing.

File: Jointify/Views/Subviews/DefaultButton.swift [hunk 3]
```diff
@@ -27,11 +27,13 @@ struct DefaultButton<ConformsToView: View>: View {
         Button(action: action, label: {
             self.label
                 .padding()
-                .background(Color.blue)
+                .background(Color.lightBlue)
                 .foregroundColor(.white)
```
Comment (lgerhardt45) [inline Jointify/Views/Subviews/DefaultButton.swift:31]: Can you also add the shade you guys proposed in the mockups, please? :-)

image

Comment (annalena94) [reply Jointify/Views/Subviews/DefaultButton.swift:31]: That is the shade I added with the custom color - I named it lightBlue ;)

Comment (lgerhardt45) [reply Jointify/Views/Subviews/DefaultButton.swift:31]: Do you have a screenshot for us? :)

Comment (annalena94) [reply Jointify/Views/Subviews/DefaultButton.swift:31]: grafik

**Comment by annalena94** [comment]
lightBlue refers to the hex shade 00C2CB we proposed in the figma wireframes

File: Jointify/Views/Subviews/DefaultButton.swift (hunk 3 above)
Comment (lgerhardt45) [reply Jointify/Views/Subviews/DefaultButton.swift:31]: Sorry, I don't mean the colour but the downwards shade of the button on the screen.

Comment (lgerhardt45) [reply Jointify/Views/Subviews/DefaultButton.swift:31]: Ich meine den Schatten, nicht den Farbton 🙈 
image

Comment (annalena94) [reply Jointify/Views/Subviews/DefaultButton.swift:31]: Ah sorry, shade in the sense of "Schatten" - I understood shade = "Farbton". Only 2 hours of sleep last to finish our paper isn't helping. Will tackle the complete UI tomorrow and Friday morning.

Comment (lgerhardt45) [reply Jointify/Views/Subviews/DefaultButton.swift:31]: Perfect, thanks a lot! 👍

Comment (annalena94) [reply Jointify/Views/Subviews/DefaultButton.swift:31]: grafik
Added the shade, please approve merge request :)

Comment (lgerhardt45) [reply Jointify/Views/Subviews/DefaultButton.swift:31]: Looks great, thanks a lot!

**Pull request merged**